lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -155,70 +187,147 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
           segment_byte_size_t,
           list(Manifest.SegmentAttribute.t())
```

Comment:
this type seems no longer valid since we pass `partial` and `name` through this argument